### PR TITLE
[Ready]Several Voice of God fixes

### LINKED ISSUE #22569: Voice of god with the new colossus organ does not work with commands
I've managed to kill and get the organ implanted, However when I issued commands to Stop, Burn, Fall and sleep. Etc, They did not work and I've tried using them in sentances and directing them at a single person and a variety of people, All commands did nothing besides make a loud noise and scare the jeebus out of some people.

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
tweak: Voice of God's Sleep lasts less than the other stuns.
add: You can also use people's jobs to single them out, instead of only names.
tweak: If multiple people share the same name/job they'll all be included, although at a reduced bonus.
tweak: Names and jobs will only be accepted if they're the first part of the command, and not in the middle, to prevent unintended focusing.
fix: Voice of God now shows speech before the emotes it causes.
fix: Special characters are no longer over-sanitized.
/:cl:

State laws command now works properly, and its cooldown has been increased to cooldown_stun (90 seconds)
Clown HONK now has cooldown_stun. It's still cooldown_meme(30 seconds) for nonclowns.
Special characters aren't double or triple sanitized anymore.
Speech will show before the voice's effects.

Side effects: 
- say() can now not sanitize text if needed.
- statelaws() now has an argument to force the borg to state unchecked laws.

<s>Closes #22570 since it includes its changes as well, credit to @ChangelingRain for those.</s> Kor is a loose cannon and merged it anyway

<s>Fixes #22569.</s> Fixes not-yet-reported issues

### CODE REVIEW

**Comment by KorPhaeron** [comment]
wait shit sorry

**Comment by XDTM** [comment]
honk

**Comment by XDTM** [comment]
do i resolve conflicts myself or do we revert the other one?

**Comment by XDTM** [comment]
that's one silly-looking commit

**Comment by XDTM** [comment]
Since it's still open i've tacked on more stuff. I'm sure this will get it merged faster!

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -60,8 +60,9 @@ var/list/department_radio_keys = list(
 
 var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 
-/mob/living/say(message, bubble_type,var/list/spans = list())
-	message = trim(copytext(sanitize(message), 1, MAX_MESSAGE_LEN))
+/mob/living/say(message, bubble_type,var/list/spans = list(), sanitize = 1)
```
Comment (coiax) [inline code/modules/mob/living/say.dm]: sanitize = TRUE

**Pull request merged**